test/models/github-test.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('assert')
+const fs = require('fs')
+const path = require('path')
+
+const GitHub = require('../../src/models/github')
+
+describe('GitHub', () => {
+  describe('getToken', () => {
+    it('returns contents of .env file', () => {
+      const tokenPath = path.join(__dirname, '..', '..', '.env')
```

Comment:
I'm not opposed to the idea of faking it somehow. The .env file being used here isn't one specific to tests, though, it's using the same .env file the app itself uses to run. One of the package.json scripts forces a .env to exist, even if it's blank, as part of initial setup.